### ISSUE TITLE
Is Source Control implemented?

### ISSUE THREAD

**Issue by noxy-k** [disguise-one/RenderStream-UE#45]
We're running source control on our UE projects and started testing the renderstream plugin (we're running an rx2 cluster).
The .json generated by the plugin doesn't seem to be going through source control and fails to update - probably because the file itself is read-only unless it gets checked out.
An easy work around is to not have the .json part of the source control but that'd mean copying this file manually to multiple machines whenever we update/work. Not feasible.

I might be completely mistaken of course - help please!

**Comment by chrisnashdisguise** [comment]
Yep this sounds like something we should fix, I'll log the issue internally so someone can take a look at it.

**Comment by noxy-k** [comment]
Thanks Chris, I would also look at the timing used to generate/update these files. At the moment some are updated using SAVE PROJECT as a trigger - which means you're doing two submits against the SVN (one for your actual changes, the second for the update the plugin just did to its files). Could be streamlined but a lesser issue.

**Comment by chrisnashdisguise** [comment]
This should be fixed in the latest releases - we now check out the schema .json before editing. Please could you test this out and let us know if it works?

**Comment by pkevin-disguise** [comment]
Haven't received feedback, but this should be fixed.

Closing.

**Comment by noxy-k** [comment]
Hi Kev,

Yes it was fixed, thanks.
